provisioning/development-osx.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: localhost
+  # remote_user: ubuntu
+  # sudo: yes
```

Comment:
Just a comment, it needs to go away (the `become: yes` bit is only in the tasks that need it).